ReactSkia/components/RSkComponentImage.cpp

```diff
@@ -3,88 +3,74 @@
 #include "include/core/SkImageFilter.h"
 #include "include/effects/SkImageFilters.h"
 
+#include "rns_shell/compositor/layers/PictureLayer.h"
+
 #include "react/renderer/components/image/ImageEventEmitter.h"
 
 #include "ReactSkia/components/RSkComponentImage.h"
 #include "ReactSkia/views/common/RSkDrawUtils.h"
 #include "ReactSkia/views/common/RSkImageUtils.h"
-#include "ReactSkia/views/common/RSkImageCacheManager.h"
 
 #include "ReactSkia/utils/RnsLog.h"
 #include "ReactSkia/utils/RnsUtils.h"
 #include "ReactSkia/views/common/RSkConversion.h"
-
+#include "ReactSkia/sdk/CurlNetworking.h"
 namespace facebook {
 namespace react {
 
 using namespace RSkDrawUtils;
 using namespace RSkImageUtils;
-using namespace RSkImageCacheManager;
-
-namespace {
-
-sk_sp<SkImage> getLocalImage(ImageSource source) {
-  if ( !source.uri.empty() && !(source.uri.substr(0, 14) == "file://assets/")) {
-    return nullptr;
-  }
-  std::string path = "./" + source.uri.substr(7);
-  RNS_PROFILE_START(getImageData)
-  sk_sp<SkImage> imageData=getImageData(path.c_str());
-  RNS_PROFILE_END(path.c_str(),getImageData)
-  if(!imageData) {
-    RNS_LOG_ERROR("Draw Image Failed :" << path);
-  }
-  #ifdef RNS_IMAGE_CACHE_USAGE_DEBUG
-      printCacheUsage();
-  #endif //RNS_IMAGECACHING_DEBUG
-  return imageData;
-}
-
-}//namespace
+using namespace ImageCacheManager;
 
 RSkComponentImage::RSkComponentImage(const ShadowView &shadowView)
-    : RSkComponent(shadowView) {}
+    : RSkComponent(shadowView) {
+      imageCacheManagerInstance_ = ImageCacheManager::RSkImageCacheManager::getImageCacheManagerInstance();
```

Comment:
ImageCacheManager namespace is already declared above. So the function call does not required using the namespace name again.